client/src/App.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './App.scss';
+import {Register} from "./components/Register";
+
+function App() {
+  return (
```

Comment:
acoording to es6 and array functions - no need ```return```

Reply:
```
consr App = () => <Register />
```

Reply:
Reworked